tools/smufl/config/valid_codes.js

```diff
@@ -196,4 +196,19 @@ module.exports = {
   vexNoteHeadMutedBreve: 'vf',
   vexNoteHeadRectBlack: 'vd4',
   vexNoteHeadRectWhite: 'vd5',
+
+  csymDiminished:null,
```

Comment:
Few stylistic notes:

- Move this block before the `vex...` blocks.
- Add a comment above the block specifying that these are chord symbols.
- Add a space between the colon and the null